core/runtime/binaryen/runtime_api/runtime_api.hpp

```diff
@@ -147,7 +147,7 @@ namespace kagome::runtime::binaryen {
       OUTCOME_TRY(res, executor_.call(*module, wasm_name, ll));
       memory->reset();
       if constexpr (!std::is_same_v<void, R>) {
-        WasmResult r{res.geti64()};
+        WasmResult r(res.geti64());
```

Comment:
Why? It's nothing improve

Reply:
We have an amiguity here, since I added constructor to WasmResult. As a struct WasmResult can initialize its members through initialization list, same time we have a constructor taking one argument. Which way should a compiler prefer?

Reply:
No, it would not happens, because class has explicit one-arg-constructor.
More usefull info look at [Habr](https://habr.com/ru/company/jugru/blog/469465/#a3)

Reply:
Hm... Seems I was not right...